console/src/Console.cpp

```diff
@@ -196,7 +196,9 @@ void initTraceAndLog() {
     megamol::core::utility::log::Log::DefaultLog.SetLevel(megamol::core::utility::log::Log::LEVEL_ALL);
     megamol::core::utility::log::Log::DefaultLog.SetEchoLevel(megamol::core::utility::log::Log::LEVEL_ALL);
     megamol::core::utility::log::Log::DefaultLog.SetOfflineMessageBufferSize(100);
-    megamol::core::utility::log::Log::DefaultLog.SetMainTarget(std::make_shared<megamol::core::utility::log::StreamTarget>(std::cout, megamol::core::utility::log::Log::LEVEL_ALL));
+    megamol::core::utility::log::Log::DefaultLog.SetMainTarget(
+        std::make_shared<megamol::core::utility::log::DefaultTarget>(megamol::core::utility::log::Log::LEVEL_ALL));
```

Comment:
Wenn kein Echo Target gestetzt wird sind alle bisherigen Lognachrichten weg, wenn später in der GUI-LogConsole das EchoTarget überschrieben wird und Reecho dann die bishereigen Logeinträge weitergeben würde.
Meine Vorschläge wären entweder, die Zeile komplett rauszunehmen, dann wird standarmäßig das OfflineTarget verwendet (bei der main.cpp ist die Zeile nicht drin). Oder damit es explizit klar wird, satt dem nullptr ein OfflineTarget übergeben.